coalib/output/ConsoleInteraction.py

```diff
@@ -763,3 +764,36 @@ def show_bears(local_bears,
     bears = inverse_dicts(local_bears, global_bears)
 
     print_bears(bears, show_description, show_params, console_printer)
+
+
+def show_language_bears_capabilities(language_bears_capabilities,
+                                     console_printer):
+    """
+    Displays what the languages bears can detect and fix
```

Comment:
also, quite unclear doc. this is only for one language, right? and shows the can fix and the can detect.
then it should be: ``Display what the bears can detect and fix.``